Library/Styles/Fonts.swift

```diff
@@ -123,7 +123,15 @@ extension UIFont {
     default: defaultSize = 17
     }
 
-    let font = UIFont.preferredFont(forTextStyle: style)
+    let font: UIFont
+    if #available(iOS 13.0, *) {
+      font = UIFont.preferredFont(
+        forTextStyle: style,
+        compatibleWith: .current
```

Comment:
Seems this is now required in order for the dynamic type changes to take effect during snapshot tests.